mongorestore/mongorestore_test.go

```diff
@@ -69,6 +70,57 @@ func TestMongorestore(t *testing.T) {
 	}
 
 	Convey("With a test MongoRestore", t, func() {
+		// As specified in TOOLS-2363, the --db and --collection options
+		// are well-defined only for restoration of a single BSON file
+		Convey("and proper warning is issued if --db and --collection are not supported", func() {
+			// Hack to make log readable when testing
+
+			// TODO: Handle all logging in main so that ParseOptions
+			// and ParseAndValidateOptions can idiomatically return
+			// testable error values
```

Comment:
We avoid leaving "TODOs" in production code. If you find yourself adding TODOs for future work, you should file a ticket so we can triage and schedule it.

Reply:
Thanks for the suggestion! Consider it done!